### ISSUE TITLE
New mobile app : time slider cropped

### ISSUE THREAD

**Issue by phiphou** [electricitymaps/electricitymaps-contrib#4975]
With the new mobile app, the time slider is a little cropped at the bottom on my GS7. 

See snapshot (full screen)

![Screenshot_20230126-171017_Electricity Maps](https://user-images.githubusercontent.com/364418/214892326-c7015973-ba9a-4212-a006-9c3157767395.jpg)

**Comment by VIKTORVAV99** [comment]
Thanks for reporting the issue!
Could you provide a little more feedback like zoom levels, screen resolution and android version so we can try and replicate this in order to find a fix?

**Comment by phiphou** [comment]
I'm on a Samsung Galaxy S7, set with a full HD resolution (1920x1080), default zoom level. 

My Android version is 8.0.0. 

![Screenshot_20230126-183528_Settings](https://user-images.githubusercontent.com/364418/214909541-9dc77005-2d43-4757-a5fc-dfc146fc91a8.jpg)

**Comment by VIKTORVAV99** [comment]
> I'm on a Samsung Galaxy S7, set with a full HD resolution (1920x1080), default zoom level. 
> 
> My Android version is 8.0.0. 
> 
> ![Screenshot_20230126-183528_Settings](https://user-images.githubusercontent.com/364418/214909541-9dc77005-2d43-4757-a5fc-dfc146fc91a8.jpg)
> 

Thanks for the additional information!

**Comment by Ankitchaudharyy** [comment]
I can work on this issue. Please assign this to me.

**Comment by VIKTORVAV99** [comment]
> I can work on this issue. Please assign this to me.

Sure thing!

**Comment by Ankitchaudharyy** [comment]
@VIKTORVAV99 @phiphou 
I tried 1920 x 1080 resolution. I am not able to reproduce the error. Help me reproducing the issue.

<img width="1438" alt="Screenshot 2023-02-09 at 7 15 29 PM" src="https://user-images.githubusercontent.com/46367738/217829907-091ef2d9-e286-4a23-add6-13d21e0f8907.png">

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99 @phiphou I tried 1920 x 1080 resolution. I am not able to reproduce the error. Help me reproducing the issue.
> 
> <img alt="Screenshot 2023-02-09 at 7 15 29 PM" width="1438" src="https://user-images.githubusercontent.com/46367738/217829907-091ef2d9-e286-4a23-add6-13d21e0f8907.png">

I don't think it's the resolution that's the problem but the screen size and orientation. Not really sure how to reproduce it the best way but try using one of the mobile layout in the Chromium dev tools.

**Comment by phiphou** [comment]
The viewport resolution on my GS7 is 360x640. I tried on chrome on my computer, no problem.
![Sans-titre-1](https://user-images.githubusercontent.com/364418/217921971-d9a137b4-5f6d-40a0-b1f8-e98cd90b7c1f.png)

**Comment by phiphou** [comment]
It may be a clue : with previous version, top notification bar disapeared when I lunch the app and reolaced with à blank bar.


![Screenshot_20230209-205109_Electricity Maps](https://user-images.githubusercontent.com/364418/217922891-682ec6a8-20f1-4be5-a60c-6ae031614590.jpg)

**Comment by tonypls** [comment]
Hey @Ankitchaudharyy how are you getting on? I can pick this up if you're busy :)

**Comment by phiphou** [comment]
There is no problem with my new smartphone (Samsung Galaxy S23)

![2023-03-11 10 21 16](https://user-images.githubusercontent.com/364418/224476305-2eede5a6-8f99-41a5-aff7-d2ddf347bfe0.jpg)

**Comment by phiphou** [comment]
Note the break label problem is still here : 

![2023-03-18-18 18](https://user-images.githubusercontent.com/364418/226122821-46a135fe-1e3a-43c3-8185-fc6ec56715f3.png)

**Comment by VIKTORVAV99** [comment]
I'll see if I can take a look at this the coming week. 👍🏼

**Comment by VIKTORVAV99** [comment]
@Ankitchaudharyy what is your progress on this matter? It would be great if we could get this resolved.

**Comment by phiphou** [comment]
I finally found the origin of the issue. I'm a bit faulty. 

On my old device (GS7), I forgot to mention I used an app, called BigFont, to make texts a little bigger on my phone.

I did not install yet on my new phone when I said there is no problem on my new GS23. With the app installed, the problem came back.

I use a 115% custom preset, so every text is 15% bigger. That cause this problem.

I was able to reproduce it on Chrome Dev Tools, see snapshot bellow.

![em](https://user-images.githubusercontent.com/364418/231230638-21661297-bcbd-4ec1-a362-5ed4376de58d.png)

A "solution" would be to replace "display data from the past" by "display history", or in french in my case "afficher les données passées" by "afficher l'historique", that fix the issue.

![histo](https://user-images.githubusercontent.com/364418/231232314-dfbdfbf4-70a2-4c25-8849-3a6f0b9f607e.png)

But I admit it's just a trick, no a real solution.

So this is not a bug anymore but a feature request.

I would understand you won't accept to change site wording for a so small issue.

Hope you'll find a more elegant solution.
